build.gradle.kts

```diff
@@ -6,11 +6,9 @@ plugins {
     `java-library`
 }
 
-val junitVersion: String by project.extra
-
 allprojects {
     group = "com.velocitypowered"
-    version = "3.2.0-SNAPSHOT"
+    version = rootProject.file("version").readLines().first()
```

Comment:
Instead of declaring the version in a separate file, I think it would be better to declare it in the `gradle.properties` file, that way you avoid creating a separate file and you also avoid this declaration of allprojects because by placing the version in the `gradle.properties` file, these values are shared throughout the project when using `project.version`

Reply:
Resolved in 39fc2805fd396d7a66bf0aa3444137714c7e7c7b